docs/articles_en/openvino-workflow-generative/inference-with-genai/inference-with-genai-on-npu.rst

```diff
@@ -231,31 +244,68 @@ Use the following code snippet to change the default settings:
          ov::AnyMap pipeline_config = { { "MAX_PROMPT_LEN",  1024 }, { "MIN_RESPONSE_LEN", 512 } };
          ov::genai::LLMPipeline pipe(model_path, "NPU", pipeline_config);
 
+In the GenAI LLM chat scenarios, the conversation history is accumulated in the context and may require
+a larger ``MAX_PROMPT_LEN`` to handle the history properly.
 
-Cache compiled models
+Performance hints
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-By caching compiled models, you may shorten the initialization time of the future pipeline
-runs. To do so, specify one of the following options in ``pipeline_config`` for NPU pipeline.
+You can configure the NPU LLM pipeline with the ``PREFILL_HINT`` and ``GENERATE_HINT`` options
+to fine-tune the performance behavior. These options impact the prompt processing (first token)
+and text generation (all other tokens) behavior respectively.
 
-NPUW_CACHE_DIR
------------------------------------------------------------------------------------------------
+``PREFILL_HINT`` -- fine-tune the prompt processing stage:
+
+* ``DYNAMIC`` (default since 2025.3) -- enables dynamic prompt execution, unblocks longer prompts.
+* ``STATIC`` -- disables dynamic prompt execution, may perform better with several precise prompt sizes.
+  Default behavior until OpenVINO 2025.3.
```

Comment:
```suggestion
* ``DYNAMIC`` (default since OpenVINO 2025.3) -- enables dynamic prompt execution, supports longer prompts.
* ``STATIC`` -- disables dynamic prompt execution, may provide better performance for specific prompt sizes.
  Default behavior before OpenVINO 2025.3.
```